python/tank/api.py

```diff
@@ -609,6 +609,28 @@ def tank_from_entity(entity_type, entity_id):
     pc = pipelineconfig_factory.from_entity(entity_type, entity_id)
     return Tank(pc)
 
+
+_current_user = None
+
+
+def set_current_user(user):
+    """
+    Sets the current Shotgun user.
```

Comment:
An small example here would be great, helps users who are not familiar with the authentication lib to understand its typical use.